core/server/lib/members/cookies.js

```diff
@@ -32,7 +32,7 @@ module.exports = function cookies(sessionSecret) {
     function removeCookie() {
         return cookie.serialize('signedin', false, {
             maxAge: 0,
-            path: '/ghost/api/v2/members/token',
+            path: '/ghost/api/v2/members',
```

Comment:
What is the reason to remove `/token` from path here? 🤔

Reply:
Because the current `/subscriptions` endpoint uses the cookie, so we need it to be accessible - this is where a `/secure` prefix would've been useful!